<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportTariffsXML</class>
 <widget class="QDialog" name="ImportTariffsXML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Импорт тарифов для договора</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="chkOnlyCodes">
     <property name="toolTip">
      <string>Не учитывать различия в наименовании</string>
     </property>
     <property name="text">
      <string>Учитывать только код услуги</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Загрузить из</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="chkFullLog">
     <property name="text">
      <string>Подробный отчет</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chkUpdateTariff">
     <property name="text">
      <string>Обновлять совпадающие тарифы</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Начать импорт</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAbort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Прервать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QTextBrowser" name="logBrowser"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtFileName"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnSelectFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtFileName</tabstop>
  <tabstop>btnSelectFile</tabstop>
  <tabstop>chkFullLog</tabstop>
  <tabstop>chkUpdateTariff</tabstop>
  <tabstop>chkOnlyCodes</tabstop>
  <tabstop>logBrowser</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>btnAbort</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
